projects/worker-cc-photoshop/actions/worker-cc-photoshop/index.js

```diff
@@ -51,11 +53,30 @@ function getAuthorization(params) {
  * @param {Object} instructions Rendition instructions
  * @returns {Object} options for Photoshop Actions Api request
  */
-async function setupPhotoshopActionsOptions(client, instructions) {
+async function setupPhotoshopActionsOptions(client, instructions, files) {
     if (!instructions || !instructions.photoshopAction) {
         throw Error("Photoshop Action url not provided");
     }
-    const options = await client.fileResolver.resolveInputsPhotoshopActionsOptions({ actions: instructions.photoshopAction });
+    
+    // workaround for action files from AEM
+    // we must download the action file and add the `.atn`
+    // extension to the file so Photoshop Service can
+    // recognize it
+    let photoshopAction = instructions.photoshopAction;
+    let ext;
+        try {
+            ext = path.extname(photoshopAction).substring(1).toLowerCase();
+        } catch (err) {
+        }
+        if (!ext) {
+            const tempActionFilename = `${uuidv4()}_temp.atn`;
+            const aioLibActionFilename = `${uuidv4()}/photoshopaction.atn`;;
```

Comment:
Double semicolon at end of line
```suggestion
            const aioLibActionFilename = `${uuidv4()}/photoshopaction.atn`;
```